hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -354,7 +363,7 @@ private XceiverClientReply sendCommandWithRetry(
         responseProto = null;
       } catch (ExecutionException e) {
         LOG.debug("Failed to execute command {} on datanode {}",
```

Comment:
This needs to be guarded by if (LOG.isDebugEnabled())

Reply:
OK

Reply:
Good point, you are right on this.